src/components/settings/SettingsSSOTab/SSOStepper.jsx

```diff
@@ -65,7 +65,7 @@ const SSOStepper = ({ enterpriseSlug, enterpriseId, enterpriseName }) => {
     // Values we want all provider configs to by default contain
     configFormData.append('enterprise_customer_uuid', enterpriseId);
     configFormData.append('enabled', true);
```

Comment:
Seems like the enabled/active setting shouldn't be forced to true -- if the config has been deactivated for some reason we probably should preserve that.  So maybe `providerConfig?enabled || true`?